### ISSUE TITLE
PTC For onboarding new team member

### ISSUE THREAD

**Issue by pagrubel** [bssw-psip/ptc-catalog#4]
# Onboarding New Team Member

> Suggest a short **name** that covers the topic of your card (like *UnitTesting*). 
> Then provide a brief description of the the goal (**target**) that a user is trying to reach with this card.

This card will cover checklist for onboarding new team member.
# User Stories
> Next, provide one or more **user stories** that explain the context of this goal.
> A user story is a statement that can be phrased as "As a _____, I want _____ so that _____."

As a mentor and/or project leader, I want to have a checklist so that I can easily help a new team member become a contributor to our project using our policies for software development.
# Card(s)

> Provide one or more cards to help a prospective user measure progress towards the goal (such as adopting a new practice or technology). A progress tracking card consists of a numbered list, where the numbers represent a set of scores (often from zero to five), and each item in the list is a qualitative description that illustrates what it means to reach that score. Zero should represent a starting state, and the highest number should represent fully achieving the goal.

Card #1 Onboarding a Novice or Student
0. No onboarding checklist in place.
1. Perform a task analysis to decide what training, resources, introductions to people with appropriate expertise etc.
2. Start writing up the checklist for training
3. Review by trying it out as if they are the novice.
4. Develop documentation for the use of dependencies for the development task.
5. Ensure resources are accessible, including permission on file structures, repositories, machines.
6. What needs to be done before they show up, order hardware.
7.  Scope the project.

# Comments
> Here you may add any additional comments to help the reader use your item in the catalog, such as definitions or links to helpful material.

# Other Remarks

> If you have any other remarks or points of discussion that you want to raise in this GitHub Issue (e.g. "Is there a better way to phrase my user story?", feel free to those here.

**Comment by pagrubel** [comment]
This was practice excersize with the IDEAS team at SC, for learning how to start a PTC in this catalog.

**Comment by jarrah42** [comment]
My understanding is that PTC items are meant to be statements of qualitative improvement rather than a series of actions. Actions are used to achieve the improvements, but are not documented in the PTC (they could be links to Github issues, etc.)

For this PTC, a target might be something like "Establish an onboarding process for new team members". 

Some example items might be:

0. No onboarding process in place
1. An initial onboarding process is documented. The process covers an established time period (pre-arrival, first day, first week, first month, etc.) and includes a checklist of activities.
2. The team onboarding process is used by supervisors for new hires.
3. The team onboarding process is approved by HR. Regular reviews of the process are undertaken.
4. All team members are familiar with, and actively participate in, the onboarding process.

**Comment by pagrubel** [comment]
Good points @jarrah42 So this was done at our meetup at SC. It seems it would be helpful if we had a better understanding of PTC.

**Comment by rinkug** [comment]
Creating a new team member on-boarding checklists for your project

0: No on-boarding checklist exists 
1. Investigate what on-boarding checklists are used by similar contemporary projects in your organization or across-organizations of similar types. 
2. Modify an existing checklist(s)/Create a new checklist(s) with a set of initial critical on-boarding tasks (spanning to a maximum of two weeks)
4. Pilot test the checklist(s) with new team members (or as a pilot dry-run) for 7 days. Obtain feedback
5. Continue pilot tests with the new team members for 14 days. Obtain feedback
6. Re-discuss feedback with team and incorporate improvements

**Comment by rinkug** [comment]
**Incorporating checklists in the off-boarding process for a team member**

**_As a project leader, I want to have a off-boarding checklist so that I can ensure that there is no loss of knowledge, work or equipment when  a team member exits the team_**

0. No on-boarding checklist exists
1. Investigate what off-boarding checklists are used by similar contemporary projects in your organization or across-organizations of similar types.
2. Modify an existing checklist(s)/Create a new checklist(s) with a set of critical off-boarding tasks 
3. Pilot test the checklist(s) as a pilot dry-run for your team with a set of employees. Obtain feedback
4. Re-discuss feedback with team and incorporate improvements and expand checklists
5. Evaluate process when a real team member exits

**Comment by jarrah42** [comment]
@rinkug is this replacing the initial PTC, or a new PTC?

**Comment by pagrubel** [comment]
It seems both are good examples to consider

**Comment by jarrah42** [comment]
This seems more like a list of tasks that would be used to develop the checklist used in item 1 of the on boarding process.

**Comment by jarrah42** [comment]
I've created a PR for this. Please comment on the PR if you have comments. I think @rinkug comments should go into a separate issue if you would think it warrants a PTC.

**Comment by rinkug** [comment]
> This seems more like a list of tasks that would be used to develop the checklist used in item 1 of the onboarding process.

The PTC I created is more for "Incorporating checklists in the on-boarding process for new team members". Its a different topic than "PTC For on-boarding new team member". Perhaps I should have created a separate issue for this topic? 

The  challenge with PTC is how to keep them high-level but at the same time how  can one pick them up and use them? I welcome ideas on how to make the more high-level.
